electrum/address_synchronizer.py

```diff
@@ -723,20 +730,39 @@ def get_addr_io(self, address):
             h = self.get_address_history(address)
             received = {}
             sent = {}
-            for tx_hash, height, *__ in h:
-                l = self.db.get_txo_addr(tx_hash, address)
-                for n, v, is_cb in l:
-                    received[tx_hash + ':%d'%n] = (height, v, is_cb)
-            for tx_hash, height, *__ in h:
+            alert_incoming = {}
+            alert_outgoing = {}
+            for tx_hash, height, tx_type, *__ in h:
+                for n, v, is_cb in self.db.get_txo_addr(tx_hash, address):
+                    key = tx_hash + ':%d' % n
+                    value = (height, v, is_cb)
+                    if tx_type == TxType.ALERT_PENDING.name:
+                        alert_incoming[key] = value
+                    elif tx_type == TxType.ALERT_CONFIRMED.name:
```

Comment:
Regarding incoming ElectrumX changes change this line to:
`elif tx_type == TxType.ALERT_CONFIRMED.name or tx_type == TxType.ALERT_RECOVERED.name:`